docs/component-stability.md

```diff
@@ -214,6 +214,9 @@ If data can be dropped/created/held at multiple distinct points in a component's
 scraping, validation, processing, etc.), it is recommended to define additional attributes to help
 diagnose the specific source of the discrepancy, or to define different signals for each.
 
+The breakdown of emitted telemetry per telemetry level (basic / normal / detailed) should follow
+the guidelines in [the Go package documentation for `configtelemetry`](config/configtelemetry/doc.go).
```

Comment:
I hope this will fix the failing links check
```suggestion
the guidelines in [the Go package documentation for `configtelemetry`](/config/configtelemetry/doc.go).
```